src/functions.php

```diff
@@ -58,6 +58,24 @@ function get_phar_compression_algorithm_extension(int $algorithm): ?string
     return $extensions[$algorithm];
 }
 
+/**
+ * @private
+ *
+ * @return <string, int>
+ */
+function get_phar_signing_algorithms(): array
```

Comment:
wow, nothing like that as native PHP function ?

Reply:
what do you mean?

Reply:
nevermind got it: no I wish that was par of the PHP phar extension as well as one or two of the functions above... but it's probably not gonna happen any time soon and didn't bother to try to push for it yet either